### ISSUE TITLE
Editor crashes on start on Fedora 23

### ISSUE THREAD

**Issue by Calinou** [AtomicGameEngine/AtomicGameEngine#708]
The Atomic editor builds successfully on Fedora 23 64-bit, but crashes when started:

Using NVIDIA driver 358.16.

```
[Thu Mar 17 13:53:12 2016] INFO: Opened log file /home/hugo/.local/share/AtomicEditor/Logs/AtomicEditor.log
[Thu Mar 17 13:53:12 2016] INFO: Created 3 worker threads
[Thu Mar 17 13:53:12 2016] INFO: Added resource path /home/hugo/Git/atomicgameengine/Artifacts/AtomicEditor/Resources/CoreData/
[Thu Mar 17 13:53:12 2016] INFO: Added resource path /home/hugo/Git/atomicgameengine/Artifacts/AtomicEditor/Resources/EditorData/
[Thu Mar 17 13:53:12 2016] DEBUG: Skipped autoload path 'Autoload' as it does not exist, check the documentation on how to set the 'resource prefix path'
[Thu Mar 17 13:53:12 2016] INFO: Set screen mode 3640x1023 windowed resizable
[Thu Mar 17 13:53:12 2016] INFO: Initialized input
[Thu Mar 17 13:53:12 2016] DEBUG: Loading resource Textures/Ramp.png
[Thu Mar 17 13:53:12 2016] DEBUG: Loading temporary resource Textures/Ramp.xml
[Thu Mar 17 13:53:12 2016] DEBUG: Loading resource Textures/Spot.png
[Thu Mar 17 13:53:12 2016] DEBUG: Loading temporary resource Textures/Spot.xml
[Thu Mar 17 13:53:12 2016] DEBUG: Loading resource Techniques/NoTexture.xml
[Thu Mar 17 13:53:12 2016] DEBUG: Loading resource RenderPaths/Forward.xml
[Thu Mar 17 13:53:12 2016] INFO: Initialized renderer
[Thu Mar 17 13:53:12 2016] INFO: Set audio mode 44100 Hz stereo interpolated
[Thu Mar 17 13:53:12 2016] INFO: Initialized engine
ALSA lib pcm.c:7905:(snd_pcm_recover) underrun occurred
ALSA lib pcm.c:7905:(snd_pcm_recover) underrun occurred
ALSA lib pcm.c:7905:(snd_pcm_recover) underrun occurred
ALSA lib pcm.c:7905:(snd_pcm_recover) underrun occurred
ALSA lib pcm.c:7905:(snd_pcm_recover) underrun occurred
ALSA lib pcm.c:7905:(snd_pcm_recover) underrun occurred
ALSA lib pcm.c:7905:(snd_pcm_recover) underrun occurred
ALSA lib pcm.c:7905:(snd_pcm_recover) underrun occurred
ALSA lib pcm.c:7905:(snd_pcm_recover) underrun occurred
ALSA lib pcm.c:7905:(snd_pcm_recover) underrun occurred
ALSA lib pcm.c:7905:(snd_pcm_recover) underrun occurred
ALSA lib pcm.c:7905:(snd_pcm_recover) underrun occurred
[0317/135312:FATAL:content_main_runner.cc(711)] Check failed: base::i18n::InitializeICU(). 
#0 0x7fbc5e3fa64e <unknown>
#1 0x7fbc5e40f9ab <unknown>
#2 0x7fbc60fa125a <unknown>
#3 0x7fbc5e31376f <unknown>
#4 0x7fbc5e31359f <unknown>
#5 0x7fbc5e2c2dc8 cef_initialize
#6 0x0000016dae62 CefInitialize()
#7 0x0000016a4089 Atomic::WebBrowserHost::WebBrowserHost()
#8 0x0000008e9399 AtomicEditor::AEEditorCommon::Start()
#9 0x0000008cfe26 AtomicEditor::AEEditorApp::Start()
#10 0x000000c3d535 Atomic::Application::Run()
#11 0x00000082c667 main
#12 0x7fbc5ac0f580 __libc_start_main
#13 0x000000862939 _start

[1]    20572 abort (core dumped)  ./AtomicEditor
```

**Comment by Elleo** [comment]
It looks like the build scripts don't copy all the needed resource files for CEF into the editor directory, if you copy the files:

Submodules/CEF/Linux/Resources/icudtl.dat
Submodules/CEF/Linux/Release/natives_blob.bin
Submodules/CEF/Linux/Release/snapshot_blob.bin

into Artifacts/AtomicEdtior/ it should then launch okay.

**Comment by Calinou** [comment]
It now starts with the files copied, but opening an example project crashes the editor.

**Comment by NateWardawg** [comment]
Thanks for the solution, I never would have figured it out on my own. Here's a quick and dirty shell script you can use. Place it in the root folder of the project. I don't think it's commit worthy though, but if anyone thinks it is, let me know and I'll commit it :)

`#!/usr/bin/env sh`

`sh Build_AtomicEditor.sh`

`cp Submodules/CEF/Linux/Resources/icudtl.dat Artifacts/AtomicEditor/`
`cp Submodules/CEF/Linux/Release/natives_blob.bin Artifacts/AtomicEditor/`
`cp Submodules/CEF/Linux/Release/snapshot_blob.bin Artifacts/AtomicEditor/`

**Comment by SajeOne** [comment]
Fixed start up crash for me on Arch Linux x86_64, thanks! :)

**Comment by JoshEngebretson** [comment]
Fixed via #712, thanks @CTrauma :+1:
